libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -23,14 +23,18 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive
     /// <summary>
     /// The Adaptive Dialog models conversation using events and events to adapt dynamicaly to changing conversation flow.
     /// </summary>
-    public class AdaptiveDialog : DialogContainer
+    public class AdaptiveDialog : DialogContainer, IDialogDependencies
     {
         [JsonProperty("$kind")]
         public const string DeclarativeType = "Microsoft.AdaptiveDialog";
 
         private const string AdaptiveKey = "adaptiveDialogState";
 
-        private readonly string changeKey = Guid.NewGuid().ToString();
+        // unique key for language generator turn property, (TURN STATE ONLY)
```

Comment:
I generally prefer starting comments with upper case and finishing with a period.

```// Unique key for language generator turn property, only used in turn state.```